src/main/java/estaru/birthdaygreetings/SendBirthdayNote.java

```diff
@@ -0,0 +1,47 @@
+package estaru.birthdaygreetings;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class SendBirthdayNote implements Reminder {
```

Comment:
There was an requirement to prepare code to send note as email but with easy way of switching to sms etc. That's why you have a Reminder interface. So implementation should tell how you send it like `EmailReminder" etc